### DISCUSSION TITLE
Cd cover art bug

### DISCUSSION THREAD

**Discussion by trybula** [rern/rAudio#1424] [Bug Reports]
Hi, there’s a strange problem with cover arts when playing cd. After loading cd cover is displayed, but as soon as I click play it disappears, and even manually adding it doesn’t work. 
![IMG_5011](https://github.com/rern/rAudio/assets/81358999/1dc8e70e-250f-4af4-8374-2ac7f2b35e89)
![IMG_5010](https://github.com/rern/rAudio/assets/81358999/cfd4a19a-3e83-45d3-93ae-fab89abf6533)

**Comment by mutlucan96** [comment]
Same here :(

**Comment by rern** [comment]
Please try: `sed -i 's/discid/\U&/' $dirbash/status-coverartonline.sh`
